services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -32,17 +32,20 @@ class ConceptFeedback < ApplicationRecord
   has_many :translation_mappings, as: :source
   has_many :english_texts, through: :translation_mappings
   has_many :translated_texts, through: :english_texts
+  has_many :gengo_jobs, through: :english_texts
   store_accessor :data, :description
 
   after_commit :clear_concept_feedbacks_cache
 
   def cache_key = "#{ALL_CONCEPT_FEEDBACKS_KEY}_#{activity_type}"
 
   def as_json(options=nil)
```

Comment:
I worry that it's a little unconventional to do that--it would still work, but `as_json(options=nil)` is such a standard rails things to do that I sort of prefer to leave it that way, even id it's a bit messier. But open to what other folks think! Maybe @brendanshean  can tiebreak?

Reply:
I would agree with Thomas' general sentiment about  `method(options=nil)` being opaque and why not just put what we're expecting, but `as_json` is documented with that method signature so I think you should leave it.